package.json

```diff
@@ -40,7 +40,7 @@
     "lodash": "4.17.11",
     "node-oauth1": "1.2.2",
     "postman-collection": "3.4.0",
-    "postman-request": "2.88.1-postman.3",
+    "postman-request": "git://github.com/postmanlabs/postman-request.git#feature/fix-formdata-order",
```

Comment:
The pull request has been approved and merged, replace this change with a proper release.